packages/react-vapor/src/components/checkbox/examples/CheckboxExamples.tsx

```diff
@@ -24,21 +26,22 @@ export class CheckboxExamples extends React.Component<any, ICheckboxWithStateSta
 
     render() {
         return (
-            <div className="mt2">
-                <h1 className="text-medium-blue mb1 bold">Checkboxes</h1>
-                <p className="m2">
+            <div>
+                <p className="pb2">
                     See{' '}
                     <a href="http://coveo.github.io/vapor/controls/#checkboxes-infos" target="_blank">
                         Usage and guidelines

```

Comment:
Is this still needed? The usage and guidelines will now be inside the "use" tab anyway.